docs/src/gcode/g-code.adoc

```diff
@@ -1086,9 +1085,9 @@ finishes accelerating to the proper speed, and can immediately begin
 cutting a good thread.
 
 .HAL Connections
-The pin 'spindle.N.at-speed' must be set or driven true for the motion to
-start. Additionally spindle.N.revs must increase by 1 for each revolution
-of the spindle and the spindle.N.index-enable pin must be connected to
+The pin `spindle._N_.at-speed` must be set or driven true for the motion to
```

Comment:
This is rendered in Asciidoc-py as
![grafik](https://user-images.githubusercontent.com/67957916/193787132-c41eaf73-f711-46b4-ac73-18ea0236292a.png)

Reply:
We agree that this should work, right? The backticks should be interpreted as monotype, not as verbatim.

Reply:
> We agree that this should work, right? The backticks should be interpreted as monotype, not as verbatim.

Looks like nested formatting, specially _italic_ has to be doubled...